### ISSUE TITLE
Even if I did not modify this part, /etc/docker/daemon.json was also modified to point the mirror to the master node.

### ISSUE THREAD

**Issue by siaimes** [microsoft/pai#5568]
Even if I did not modify this part, /etc/docker/daemon.json was also modified to point the mirror to the master node.

```
#######################################################################
#                    OpenPAI Customized Settings                      #
#######################################################################
# enable_hived_scheduler: true
# enable_docker_cache: false
# docker_cache_storage_backend: "azure" # or "filesystem"
# docker_cache_azure_account_name: ""
# docker_cache_azure_account_key: ""
# docker_cache_azure_container_name: "dockerregistry"
# docker_cache_fs_mount_path: "/var/lib/registry"
# docker_cache_remote_url: "https://registry-1.docker.io"
# docker_cache_htpasswd: ""
# enable_marketplace: "false"
```

_Originally posted by @siaimes in https://github.com/microsoft/pai/issues/5445#issuecomment-827291642_

**Comment by siaimes** [comment]
When calling this command, the conditional check that ensure `enable_docker_cache=True` is missing:

```bash
ansible-playbook -i ${HOME}/pai-deploy/cluster-cfg/hosts.yml docker-cache-config-distribute.yml || exit $?
```
[quick-start-kubespray.sh#L60](https://github.com/microsoft/pai/blob/68d29eff32db171b46293b6dfc066d671e565e13/contrib/kubespray/quick-start-kubespray.sh#L60)

**Comment by hzy46** [comment]
For a workaround, you can just remove this line.

I think it is caused by https://github.com/microsoft/pai/blob/68d29eff32db171b46293b6dfc066d671e565e13/contrib/kubespray/docker-cache-config-distribute.yml#L8 .

**Comment by siaimes** [comment]
> For a workaround, you can just remove this line.
> 
> I think it is caused by
> 
> https://github.com/microsoft/pai/blob/68d29eff32db171b46293b6dfc066d671e565e13/contrib/kubespray/docker-cache-config-distribute.yml#L8
> 
> .

Thank you for your reply. Remove this line is my current practice. I just report this bug so that contributors can fix it.

**Comment by siaimes** [comment]
fix in #5574
